src/mscorlib/ref/mscorlib.csproj

```diff
@@ -23,7 +23,6 @@
     
     <ErrorReport>prompt</ErrorReport>
     <Optimize Condition="'$(Optimize)' == ''">true</Optimize>
-    <GenerateNativeVersionInfo Condition="'$(OsEnvironment)'=='Windows_NT'">true</GenerateNativeVersionInfo>
```

Comment:
Please do confirm that this does not affect any native binary incase they accidentally took dependency on this.